src/libsyntax/ast_util.rs

```diff
@@ -328,17 +329,28 @@ pub fn operator_prec(op: ast::BinOp) -> uint {
       BiBitAnd                  =>  8u,
       BiBitXor                  =>  7u,
       BiBitOr                   =>  6u,
+      // '..' sits here with 5
       BiLt | BiLe | BiGe | BiGt | BiEq | BiNe => 3u,
       BiAnd                     =>  2u,
       BiOr                      =>  1u
   }
 }
 
+/// Precedence of the prefix operators '!', '-', '&', '*', '~' etc.
+/// Does not include the prefix form of '..'
+#[allow(non_upper_case_globals)]
+pub static prefix_prec: uint = 13u;
```

Comment:
Nit, and somewhat pre-existing, but (a) this should be a `const`, not a `static`, and (b) it'd be nice if it followed the CAPITALIZATION convention. I found it very confusing when reading the parser -- it was hard to distinguish "fixed" precedence levels like `prefix_prec` from "varying" ones like `min_prec`. Can you rename it? Thanks.